bot/exts/evergreen/cowsay.py

```diff
@@ -0,0 +1,64 @@
+from random import choice
```

Comment:
Can we directly just do `import random`. 

Just seeing `choice()` in the code can sometimes be confusing, keeping it `random.choice()` would make it clear.